modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/dcrm/api/util/OAuthDCRMConstants.java

```diff
@@ -46,6 +46,23 @@ public class OAuthDCRMConstants {
     public static final String GRANT_TYPE_AUTHORIZATION_CODE = "authorization_code";
     public static final String GRANT_TYPE_PASSWORD = "password";
 
+    public static final String TOKEN_AUTH_METHOD =  "token_endpoint_auth_method";
+    public static final String TOKEN_AUTH_SIGNATURE_ALGORITHM = "token_endpoint_auth_signing_alg";
+    public static final String SECTOR_IDENTIFIER_URI = "sector_identifier_uri";
+    public static final String ID_TOKEN_SIGNATURE_ALGORITHM = "id_token_signed_response_alg";
+    public static final String ID_TOKEN_ENCRYPTION_ALGORITHM = "id_token_encrypted_response_alg";
+    public static final String ID_TOKEN_ENCRYPTION_METHOD = "id_token_encrypted_response_enc";
+    public static final String AUTH_RESPONSE_SIGNATURE_ALGORITHM = "authorization_signed_response_alg";
```

Comment:
Didn't we discuss to remove this?

Reply:
removed from code... removed from constants as well